electron/index.html

```diff
@@ -61,7 +61,7 @@ <h3 class="beleren link">{deck.pool_name}</h3>
         <div id="deck-list" rv-show="list_selected">
           <h3 class="beleren link" onclick="unpopulateDecklist()">Back to decklists</h3>
           <h1> <i class="fas fa-chevron-left"></i> { selected_list_name }</h1>
-          <div class="card" rv-each-stat="selected_list | decklistSort" rv-card_color="stat.colors">
+          <div class="card" rv-each-stat="selected_list | decklistMergeDuplicates | decklistSort" rv-card_color="stat.colors">
```

Comment:
Huh, I didn't even know you _could_ apply multiple formatters like this. awesome!